### PR TITLE
style: use Figma icons

### LINKED ISSUE #733: Exchange all icons for those of Figma
## What is the feature about

`@mui/icons-material` is used in most places whereas we should be using as many icons as possible from Figma in order to align our design with mobile.

## The list of requirements

- Search for `@mui/icons-material`, replacing their relevant counterpart from Figma.

## Links

- [Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=0%3A1)

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves #733

## How this PR fixes it

All icons used on Figma (that are not from MUI) now replace those which were from MUI

### Icon issues

- Safe apps icons: the bordered-bookmark icon fills, despite only specifying a `stroke`. The standard icon, in `border` colour, is currently used in it's place. The share icon has a filled circle on the left as well?

image
image

- The execution rocket `fill`s, whereas it should only `stroke`

image
image

## How to test it

Navigate the Safe and observe the new icons, matching that of the Figma designs (and no visual inconsistencies).

### CODE REVIEW

**Review by katspaugh — APPROVED** [review]
Massive

File: src/components/create-safe/steps/OwnerPolicyStep.tsx [hunk 1]
```diff
@@ -104,8 +105,12 @@ const OwnerPolicyStep = ({ params, onSubmit, setStep, onBack }: Props): ReactEle
               <OwnerRow key={field.id} index={index} remove={remove} />
             ))}
 
-            <Button onClick={addOwner} sx={{ fontWeight: 'normal' }}>
-              + Add another owner
+            <Button
+              onClick={addOwner}
+              sx={{ fontWeight: 'normal' }}
+              startIcon={<SvgIcon component={AddIcon} fontSize="small" />}
```
Comment (DiogoSoaress) [inline src/components/create-safe/steps/OwnerPolicyStep.tsx:111]:
```suggestion
              startIcon={<SvgIcon component={AddIcon} fontSize="small" inheritViewBox />}
```

**Review by DiogoSoaress — APPROVED** [review]
Just found a small inconsistency. Great work here!!

**Comment by liliya-soroka** [comment]
@iamacook , could we merge it without the QA on PR? I will check the main ticket together with functional testing on https://web-core.pages.dev/rin:0x6cf158C37354d8da5396fb6d3e965318CCf119c1/home

**Comment by iamacook** [comment]
> @iamacook , could we merge it without the QA on PR?

Sure. I've just discussed the icon issues with @TanyaEfremova so please be aware that the following icons will be updated (likely tomorrow): NOT bookmarked Apps, share Apps and execution rocket.

**Comment by TanyaEfremova** [comment]
@iamacook as discussed, here are the icons, as far as I understand 16px are used. These should be correct.  
icon-rocket
icon-share

**Comment by iamacook** [comment]
> @iamacook as discussed, here are the icon

Thank you! I'll create another PR to include them in a bit.

**Pull request merged**